examples/benchmark/deserialize_benchmark.py

```diff
@@ -0,0 +1,172 @@
+import torch
+import shutil
+import os
+import time
+import subprocess
+from pathlib import Path
+from tensorizer import TensorDeserializer
+from tensorizer.utils import convert_bytes, get_mem_usage
+import numpy as np
+
+# disable missing keys and unexpected key warnings
+os.environ["TRANSFORMERS_VERBOSITY"] = "error"
+
+# Improve safetensors performance
+os.environ["SAFETENSORS_FAST_GPU"] = "1"
+
+from accelerate import init_empty_weights
+from transformers import AutoModelForCausalLM, AutoTokenizer, AutoConfig
+
+
+MODEL_PATH = Path(os.environ.get("MODEL_PATH", f"./models"))
+TZR_PATH = MODEL_PATH / "model.tensors"
+HF_PATH = MODEL_PATH / "hf"
+RES_PATH = Path(os.environ.get("RES_PATH", "./results"))
+NUM_TRIALS = int(os.environ.get("NUM_TRIALS", 1))
+SKIP_HF = os.environ.get("SKIP_HF", False)
+SKIP_TZR = os.environ.get("SKIP_TZR", False)
+SKIP_ST = os.environ.get("SKIP_ST", False)
+SKIP_INFERENCE = os.environ.get("SKIP_INFERENCE", False)
+CURL_PATH = shutil.which("curl")
+
+
+RES_PATH.mkdir(parents=True, exist_ok=True)
+config = AutoConfig.from_pretrained(HF_PATH)
+
+DEVICE = torch.device("cuda:0")
+
+
+def run_curl(output: str = "/dev/null"):
+    """
+    Used to compare pod's network speed to tensorizer's speed when using S3/HTTP
+    """
+
+    http_uri = "http://tensorized.accel-object.ord1.coreweave.com/EleutherAI/gpt-j-6B/fp16/model.tensors"
```

Comment:
Can we make this URL a parameter we can change?

Reply:
This `run_curl()` function is left over from when I was doing benchmarks for loading from S3. It isn't being used in this code anymore so I should have removed it, but I'll remove it now.